src/org/opendatakit/briefcase/export/Csv.java

```diff
@@ -60,11 +60,9 @@ static Csv repeat(FormDefinition formDefinition, Model groupModel, ExportConfigu
         .orElse(stripIllegalChars(formDefinition.getFormName()));
     String suffix = groupModel.getName();
     Model current = groupModel;
-    while (current.hasParent()) {
+    while (current.hasParent() && current.getParent().hasParent() && current.getParent().getParent().getName() != null) {
```

Comment:
Was going to ask for a comment and maybe a method and then went to the next commit and found peace. 😄